cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-eastus-variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "east_resource_group_name" {
+  type = "string"
+}
+
+variable "east_resource_group_location" {
+  type    = "string"
+}
+
+variable "gitops_east_path" {
+  type = "string"
+}
+<<<<<<< HEAD
```

Comment:
Fix merge conflict.

Reply:
Fixed merge conflict

Reply:
Still see some conflicts. Will resolve it tomorrow